### PR TITLE
[Javascript]: nanoexpress - Add ENV variables

### PR DESCRIPTION
**Pull request by dalisoft**

### CODE REVIEW

File: javascript/nanoexpress/config.yaml [hunk 1]
```diff
@@ -2,6 +2,9 @@ framework:
   github: nanoexpress/nanoexpress
   version: 1.1
 
+environment:
+  EXPERIMENTAL_FASTCALL: 1
```
Comment (waghanza) [inline javascript/nanoexpress/config.yaml]: what is this for @dalisoft ?

Comment (dalisoft) [reply javascript/nanoexpress/config.yaml]: It's for fix from performance drop. You can read about this bit more from [here](https://github.com/uNetworking/uWebSockets.js/releases/tag/v17.2.0)

**Comment by dalisoft** [comment]
Can i ask why this PR is closed? Thanks

**Comment by waghanza** [comment]
Ah sorry

**Comment by waghanza** [comment]
Are your `Pr`s ready ?

**Comment by dalisoft** [comment]
I think, Yes

**Pull request merged**